hawc/apps/lit/api.py

```diff
@@ -55,6 +56,27 @@ def reference_tags(self, request, pk):
         df = models.ReferenceTags.objects.as_dataframe(instance.id)
         return Response(df)
 
+    @decorators.detail_route(
+        methods=("get", "post"), url_path="references-download", renderer_classes=PandasRenderers
+    )
+    def references_download(self, request, pk):
+        """
+        Get all references in an assessment.
+        """
+
+        self.set_legacy_attr(pk)
+
+        self.tags = models.ReferenceFilterTag.get_all_tags(self.assessment.id, json_encode=False)
```

Comment:
we can just set this to a local variable; no need to bind to the class (change `self.tags =` to `tags = `)

Reply:
Done